build/archives/BUILD.gn

```diff
@@ -127,23 +127,41 @@ zip_bundle("flutter_patched_sdk") {
   ]
 }
 
+generated_file("dart_sdk_entitlement_config") {
+  outputs = [ "$target_gen_dir/dart_sdk_entitlements.txt" ]
+
+  contents = [
```

Comment:
This seems fine since the Dart SDK is a prebuilt archive, but it might be good to add in a comment that the rule for the Dart SDK is here https://github.com/flutter/engine/blob/main/BUILD.gn#L61.

Reply:
Thanks for reviewing! Added comment.